packages/kbn-es-query/src/es_query/build_es_query.ts

```diff
@@ -44,7 +45,7 @@ function removeMatchAll<T>(filters: T[]) {
  */
 export function buildEsQuery(
   indexPattern: DataViewBase | undefined,
-  queries: Query | Query[],
+  queries: Query | AggregateQuery | Array<Query | AggregateQuery>,
```

Comment:
it may be useful to create a top-level type like:

```
type AnyQuery = Query | AggregateQuery
```

And reference that everywhere instead.